src/MeetingsSDKAdapter.js

```diff
@@ -295,13 +306,20 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
    * @returns {Observable.<MeetingControlDisplay>}
    * @memberof MeetingJSONAdapter
    */
-  muteVideoControl(ID) {
+  videoControl(ID) {
     const sdkMeeting = this.fetchMeeting(ID);
     const muted = {
-      ID: MUTE_VIDEO_CONTROL,
+      ID: VIDEO_CONTROL,
+      icon: 'camera-muted',
+      tooltip: 'Unmute',
+      state: MeetingControlState.ACTIVE,
+      text: null,
+    };
+    const unmuted = {
+      ID: VIDEO_CONTROL,
       icon: 'camera',
```

Comment:
I just re-checked the UX design, for both, mute and unmute states, the icon should be `camera-muted`